cmd/pktvisord/main.cpp

```diff
@@ -100,6 +100,14 @@ static const char USAGE[] =
     Prometheus Options:
       --prometheus                          Ignored, Prometheus output always enabled (left for backwards compatibility)
       --prom-instance ID                    Optionally set the 'instance' label to given ID
+    Opentelemetry Options
+      --opentelemetry                       Enable Opentelemetry OTLP exporter over HTTP
```

Comment:
since everything else here is `otel` i suggest we keep this `--otel` too. also in description let's use `HTTP(S)`

Reply:
done